.github/actions/setup-poetry/action.yaml

```diff
@@ -8,17 +8,13 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - run: |
-        pip install --user pipx
-        pipx ensurepath
-      shell: bash
     - if: ${{ inputs.poetry-version == 'latest' }}
       run: |
-        pipx install poetry
+        pip install poetry
```

Comment:
@PatrickDinh is this parametrized so it only uses pip when its apple silicon? I think we should still stick to pipx for other pipelines - arm silicon will be adding pipx at some point so its rather an exceptional case